### ISSUE TITLE
Stretched Fixed Mode - Integer Scaling Issue

### ISSUE THREAD

**Issue by KevinArriola** [runelite/runelite#4504]
Screen flickering and constant resizing when enabling integer scaling on stretched fixed mode, with the keep aspect ratio option enabled. Integer scaling on its own is producing an image half the normal size. Keep aspect ratio without integer scaling works correctly. Issue unaffected by hardware acceleration.

Screen recording: https://youtu.be/3evA-Thx3x4

Runelite version 1.4.8.1
Windows 10
Intel Graphics HD 500

[logs.zip](https://github.com/runelite/runelite/files/2230505/logs.zip)

**Comment by deathbeam** [comment]
Hmm this is interesting, so the screen just stays in that weird corrupted mode after turning on integer scaling?

**Comment by robmonte** [comment]
Runelite V1.4.9.1
OSX 10.13.6

@deathbeam An issue with integer scaling for me also, the option is no longer working as expected. It is keeping the game at its default size instead of resizing it now but it was working a few weeks ago on the same computer.

**Comment by raku-cat** [comment]
I get a similar issue on linux. On my 1600x900 monitor it loops trying to resize, if i open a side menu it stops and just goes to the default small size, turning off keep aspect ratio causes the game to be very short(height) but stretched along the screen, moving the window to my 1920x1080 monitor causes the game to resize to 2x and cause no issues.

**Comment by Nightfirecat** [comment]
Is this still an issue? The video in the original issue report has gone missing, and I have not heard of this issue being reported since the last post.

**Comment by robmonte** [comment]
@Nightfirecat Yes, Integer Scaling is still broken and non-functional on MacOS. The Stretched Mode plugin as a whole seems to at least work in some regard now, such as properly fitting the game into the window (it still randomly will glitch sometimes though it usually goes away after toggling off/on).

However, Integer Scaling itself seems to now do nothing at all. Turning on this setting does nothing for performance. I would say this is a pretty significant feature to be broken on MacOS due to the fact that Apple's machines tend to have very high resolution displays. The game absolutely chugs dropping to sub-20 FPS when idling. This is on a 2018 15" Macbook Pro.

**Comment by Nightfirecat** [comment]
What is your screen resolution? Integer scaling requires a pretty significant vertical resolution to have any effect as most people using it do not have large enough displays to reach even 2x size.

**Comment by robmonte** [comment]
@Nightfirecat 2880x1800. There is definitely enough room for 4x game windows. You can find past issues where I've documented the same occurrences so you can see I'm consistent. https://github.com/runelite/runelite/issues/6022 and https://github.com/runelite/runelite/issues/6490

&nbsp;

Here are new screenshots taken today. The first is untouched with the stretched plugin OFF:

![image](https://user-images.githubusercontent.com/17119716/78328579-23adb000-7545-11ea-895a-e293b029561b.png)

&nbsp;

Here is me showing there's ample room available for a 2x integer scale to apply:

![image](https://user-images.githubusercontent.com/17119716/78328644-463fc900-7545-11ea-81f6-b70ef9c00235.png)

&nbsp;

If you look at the second issue I mentioned, the warped stretching is _no longer occurring_ (what I was referencing in my previous comment about working in some regard). However, it's clear that the game even now is still trying to render at full resolution regardless of integer scaling being checked or not. The login screen gives me like 5 FPS.
